src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -79,10 +84,12 @@ export const RoomModalContent: React.FC<RoomModalContentProps> = ({
   const roomEvents = useMemo(() => {
     if (!venueEvents) return [];
 
-    return venueEvents.filter(
-      (event) =>
-        event.room === room.title && isBefore(Date.now(), eventEndTime(event))
-    );
+    return venueEvents
+      .filter(
+        (event) =>
+          event.room === room.title && isBefore(Date.now(), eventEndTime(event))
```

Comment:
I think there might be a helper in `utils/events` that can replace `isBefore(Date.now(), eventEndTime(event)`

Reply:
replaced with `isEventLiveOrFuture` from `utils/events` 8fbc3f39505799cf9aba6470dac21321f5f7db56